torch/_inductor/ir.py

```diff
@@ -250,9 +250,10 @@ def ir_node_to_tensor(x, guard_shape=True):
     device = x.get_device()
     size = convert_shape_to_symint(size)
     stride = convert_shape_to_symint(stride)
-    t = torch.empty_strided(
-        size=size, stride=stride, dtype=dtype, device=device
-    ).zero_()
+    with V.graph.sizevars.shape_env.suppress_guards():
```

Comment:
just to confirm - we fixed the suppression issues, right ?

Reply:
Yeah, it was fixed in https://github.com/pytorch/pytorch/pull/126210